src/webviews/src/Step.tsx

```diff
@@ -35,38 +35,21 @@ export const Step: FunctionComponent<StepProps> = props => {
   }
 
   return (
-    <div className="flex mb-6">
-      <div
-        className="flex-none w-5 pt-2 text-right"
-        style={{
-          paddingTop: 13,
-        }}
-      >
-        {props.index + 1 - 2}.
+    <div className="p-4 rounded border-solid border border-[color:var(--divider-background)]">
```

Comment:
I wasn't able to see these views given the crash issues I was having—I saw it briefly and it looked good to me. Will check again when I can see them. Or feel free to send over a screenshot.